src/schemas/json/bitrise.json

```diff
@@ -0,0 +1,445 @@
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "BashStepToolkitModel": {
+      "properties": {
+        "entry_file": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "BitriseDataModel": {
+      "required": ["format_version", "project_type"],
+      "properties": {
+        "format_version": {
+          "type": "string"
+        },
+        "default_step_lib_source": {
+          "type": "string"
+        },
+        "project_type": {
+          "type": "string"
+        },
+        "title": {
+          "type": "string"
+        },
+        "summary": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "app": {
+          "$ref": "#/definitions/AppModel"
+        },
+        "meta": {
+          "patternProperties": {
+            ".*": {
+              "additionalProperties": true
+            }
+          },
+          "type": "object"
+        },
+        "trigger_map": {
+          "items": {
+            "$ref": "#/definitions/TriggerMapItemModel"
+          },
+          "type": "array"
+        },
+        "pipelines": {
+          "patternProperties": {
+            ".*": {
+              "$ref": "#/definitions/PipelineModel"
+            }
+          },
+          "type": "object"
+        },
+        "stages": {
+          "patternProperties": {
+            ".*": {
+              "$ref": "#/definitions/StageModel"
+            }
+          },
+          "type": "object"
+        },
+        "workflows": {
+          "patternProperties": {
+            ".*": {
+              "$ref": "#/definitions/WorkflowModel"
+            }
+          },
+          "type": "object"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "BrewDepModel": {
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "bin_name": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "CheckOnlyDepModel": {
+      "properties": {
+        "name": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "DependencyModel": {
+      "properties": {
+        "manager": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "DepsModel": {
+      "properties": {
+        "brew": {
+          "items": {
+            "$ref": "#/definitions/BrewDepModel"
+          },
+          "type": "array"
+        },
+        "apt_get": {
+          "items": {
+            "$ref": "#/definitions/AptGetDepModel"
+          },
+          "type": "array"
+        },
+        "check_only": {
+          "items": {
```

Comment:
this accepts only a single value right?

Reply:
It's defined as a slice though: https://github.com/bitrise-io/stepman/blob/master/models/models.go#L52 
Where is that singular value being enforced?